packages/block-editor/src/components/default-block-appender/style.scss

```diff
@@ -51,21 +51,6 @@
 	line-height: $editor-line-height;
 }
 
-// Basic look for sibling inserter, and side-inserter.
-.block-editor-inserter__toggle.components-button {
```

Comment:
This was being applied to all inserters and not just the appender.
What we really want is to apply it to the inserters in the canvas.